plugins/@grouparoo/hubspot/src/lib/export-objects/destinationMappingOptions.ts

```diff
@@ -97,23 +102,27 @@ export const getObjectFields = (
   important?: boolean;
 }> => {
   const properties = customObject.getProperties();
+  const requiredFields = customObject.getRequiredProperties();
   const out = [];
   for (const property of properties) {
     if (
       property["name"] !== primaryKey &&
-      !property["archived"] &&
-      !property["calculated"]
+      !property.archived &&
+      !property.calculated &&
+      !property.modificationMetadata.readOnlyValue === true &&
```

Comment:
this look weird to me.
does `!property.modificationMetadata.readOnlyValue` work?
same below with `!property.modificationMetadata.readOnlyOptions`